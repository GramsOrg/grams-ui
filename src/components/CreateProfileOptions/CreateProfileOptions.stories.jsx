import React from 'react';

import { action } from "@storybook/addon-actions";

import CreateProfileOptions from './CreateProfileOptions.tsx';

export default {
  title: 'Onboarding/CreateProfileOptions',
  component: CreateProfileOptions,
  parameters: {
    componentSubtitle: 'Component description',
    storyshots: { disable: false },
  },
  argTypes: {
    dir: {
      control: 'radio',
      options: ['ltr', 'rtl']
    },
    inverted: {
      control: 'boolean'
    }
  }
};

const Template = ({dir, inverted, ...props}) => {
  return (
    <CreateProfileOptions
      dir={dir}
      inverted={inverted}
      {...props}
    />
  );
};

export const Default = Template.bind({});
Default.args = {
  dir: 'ltr',
  inverted: false
}

export const RTL = Template.bind({});
RTL.args = {
  dir: 'rtl',
  inverted: false
}
