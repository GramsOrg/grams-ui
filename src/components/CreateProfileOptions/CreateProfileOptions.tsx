import React from 'react';

import './CreateProfileOptions.css';

export interface ICreateProfileOptionsProps {

  /**
   * Defines the direction of the component
   */
  dir?: 'ltr' | 'rtl';

  /**
   * Toggles between light and dark modes
   */
  inverted?: boolean;
}

const defaultProps = {
  dir: 'ltr',
  inverted: false
};

const CreateProfileOptions = (props: ICreateProfileOptionsProps) => {

  return (
    <>
    </>
  );
};

CreateProfileOptions.defaultProps = defaultProps;

export default CreateProfileOptions;
