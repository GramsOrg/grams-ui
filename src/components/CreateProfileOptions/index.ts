import CreateProfileOptions from './CreateProfileOptions';
export default CreateProfileOptions;
