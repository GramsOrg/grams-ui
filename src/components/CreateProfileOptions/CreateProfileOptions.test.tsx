/**
 * @jest-environment jsdom
 */

import React from 'react';
import { describe, expect, test } from '@jest/globals';
import { render } from '@testing-library/react';

import CreateProfileOptions from './CreateProfileOptions';

describe('CreateProfileOptions Component', () => {
  test('it should match the snapshot', () => {
    const { asFragment } = render(<CreateProfileOptions />);
    expect(asFragment()).toMatchSnapshot();
  });
});
