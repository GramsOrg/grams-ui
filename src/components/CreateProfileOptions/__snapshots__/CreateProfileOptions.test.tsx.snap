// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateProfileOptions Component it should match the snapshot 1`] = `<DocumentFragment />`;
